scripts/servermanager.sh

```diff
@@ -0,0 +1,167 @@
+        pp --info "> Setting Multi-Core-Enhancements to enabled\n"
+        START_OPTIONS="$START_OPTIONS -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS"
+    fi
+    if [[ -n $WEBHOOK_ENABLED ]] && [[ $WEBHOOK_ENABLED == "true" ]]; then
+        send_webhook_notification "$WEBHOOK_START_TITLE" "$WEBHOOK_START_DESCRIPTION" "$WEBHOOK_START_COLOR"
+    fi
+    ./PalServer.sh "$START_OPTIONS"
+}
+
+function stop_server() {
+    pp --warning ">>> Stopping server...\n\n"
+
+    rc 'broadcast Server_Shutdown_requested' "> Broadcasting server shutdown request..."
+    rc 'broadcast Saving...'
+    rc 'save'
+    rc 'broadcast Done...' "> Broadcasting server shutdown request..."
+
+
+    pp --info "> Creating server backup before stopping server...\n"
+
+    backup --create
+
+	kill -SIGTERM "$(pidof PalServer-Linux-Test)"
+	tail --pid="$(pidof PalServer-Linux-Test)" -f 2>/dev/null
+
+    if [[ -n $WEBHOOK_ENABLED ]] && [[ $WEBHOOK_ENABLED == "true" ]]; then
+        send_webhook_notification "$WEBHOOK_STOP_TITLE" "$WEBHOOK_STOP_DESCRIPTION" "$WEBHOOK_STOP_COLOR"
+    fi
+	
+    exit 143;
+}
+
+function install_server() {
+    # Force a fresh install of all
+    pp --success "\n>>> Doing a fresh install of the gameserver...\n\n"
+    if [[ -n $WEBHOOK_ENABLED ]] && [[ $WEBHOOK_ENABLED == "true" ]]; then
+        send_webhook_notification "Installing server" "Server is beeing installed" "$WEBHOOK_INFO_COLOR"
+    fi
+    ${steamcmd_dir}/steamcmd.sh +force_install_dir "$GAME_ROOT" +login anonymous +app_update 2394010 validate +quit
+    pp --success ">>> Done installing the gameserver.\n"
+}
+
+function update_server() {
+    # Force an update and validation
+    if [[ -n $STEAMCMD_VALIDATE_FILES ]] && [[ $STEAMCMD_VALIDATE_FILES == "true" ]]; then
+        pp --success "\n>>> Doing an update and validate of the gameserver files...\n\n"
+        if [[ -n $WEBHOOK_ENABLED ]] && [[ $WEBHOOK_ENABLED == "true" ]]; then
+            send_webhook_notification "Updating server" "Server is beeing updated and validated" "$WEBHOOK_INFO_COLOR"
+        fi
+        ${steamcmd_dir}/steamcmd.sh +force_install_dir "$GAME_ROOT" +login anonymous +app_update 2394010 validate +quit
+        pp --success ">>> Done updating and validating the gameserver files.\n"
+
+    else
+        pp --success "\n>>> Doing an update of the gameserver files...\n\n"
+        if [[ -n $WEBHOOK_ENABLED ]] && [[ $WEBHOOK_ENABLED == "true" ]]; then
+            send_webhook_notification "Updating server" "Server is beeing updated" "$WEBHOOK_INFO_COLOR"
+        fi
+        ${steamcmd_dir}/steamcmd.sh +force_install_dir "$GAME_ROOT" +login anonymous +app_update 2394010 +quit
+        pp --success ">>> Done updating the gameserver files.\n"
+    fi
+}
+
+
+### Signal Handling (Handlers & Traps)
+
+# Handler for SIGTERM from docker stop command
+function termHandler() {
+    stopServer
+}
+
+function start_handlers() {
```

Comment:
This function doesnt exist either, its litteraly a function around rainbow-puke to establish 1 line, that was before that in the right place. Why does this exist?